<ui version="4.0" >
 <class>SelectTaskDialog</class>
 <widget class="QDialog" name="SelectTaskDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select Task...</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Filter:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QToolButton" name="buttonClearFilter" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>...</string>
     </property>
     <property name="autoRaise" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QLineEdit" name="lineEditFilter" />
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QTreeView" name="treeView" />
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditFilter</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>treeView</tabstop>
  <tabstop>buttonClearFilter</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectTaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>300</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel" >
     <x>285</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectTaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel" >
     <x>318</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonClearFilter</sender>
   <signal>clicked()</signal>
   <receiver>lineEditFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>24</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>80</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
